blosc/blosc2.c

```diff
@@ -1017,9 +1016,6 @@ static int blosc_c(struct thread_context* thread_context, int32_t bsize,
   }
   neblock = bsize / nstreams;
   for (j = 0; j < nstreams; j++) {
-    if (instr_codec) {
-      blosc_set_timestamp(&last);
-    }
```

Comment:
The time instrumentation above is necessary, please re-add it.

Reply:
solved at f2fdffa71814dd1cbda6d3388e551c7e61214f64